R/process.R

```diff
@@ -151,6 +127,7 @@ dataset_process <- function(filename_data_raw,
   }
 
   # Where missing, fill variables in traits table with values from locations
+  # Currently overwriting dataset-level column metadata -- NEED FIX
```

Comment:
Isn't that what is meant to happen? If there is location-specific metadata, it should overwrite the dataset-level value for observations at that location.

Reply:
@ehwenk It's overwriting the dataset-level metadata which is read in from a column. I thought the idea was that if it's read in from a column, it's likely the final value for that observation.

Reply:
@yangsophieee Oh I see, I thought you just meant it was overriding a fixed value. I see the problem though - because you have a add code that has different actions for fixed values vs values from columns. Maybe only allow the location-level or trait-level value to be used if the dataset value is `fixed` (i.e. not a column name)?